app/iot/devices.py

```diff
@@ -1,64 +1,70 @@
-import time
+import asyncio
 
 from .message import MessageType
```

Comment:
Ensure that the MessageType import is correct and that the MessageType class or enum has a 'name' attribute, as it is used in the send_message method.